app/controllers/TwilioController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class TwilioController extends BaseController
+{
+
+	public function respond()
+	{
+		$phoneCall				 = new \models\PhoneCall();
+		$phoneCall->phone_number = Input::get('phone_number');
+		$phoneCall->save();
+
+		$response = new Services_Twilio_Twiml();
+		$response->say('Calling, wait a minute');
+		$response->dial(getenv('TWILIO_API_NUMBER'));
+
+		print $response;
```

Comment:
Laravel has functionality to work with [Responces](https://laravel.com/docs/5.1/responses)